### ISSUE TITLE
Error building 'GR'

### ISSUE THREAD

**Issue by KingNook** [johnryantaylor/FDSE#2]
When running `gravitycurrent.jl`, Julia raises 

```
ERROR: Error building `GR`:
[ Info: Downloading pre-compiled GR 0.52.0 Windows binary
ERROR: LoadError: IOError: could not spawn `'C:\Users\nookh\AppData\Local\Programs\Julia-1.9.2\bin\..\libexec/7z' x downloads/gr-0.52.0-Windows-x86_64.tar.gz -y`: no such file or directory (ENOENT)
```

**Comment by mlin706** [comment]
Copying 7z.exe from _Julia-1.9.2\\libexec\\julia_ to _Julia-1.9.2\\libexec_ fixes the issue

**Comment by johnryantaylor** [comment]
Good to know, thanks!On Jul 22, 2023, at 12:06 PM, Michael ***@***.***> wrote:﻿
Closed #2 as completed.
